.gitignore

```diff
@@ -2,13 +2,13 @@ _site
 
 .jekyll-cache
 
-CNAME
+/CNAME
```

Comment:
@Zain-Muiz are these ignore statements not working as-is?

Reply:
I see your description in the issue raised now. 

We actually do want to track these in the repo. So, it's actually a good thing that they are being ignored. Do you mind updating this PR to have these three files unignored?